scripts/main/build.js

```diff
@@ -176,11 +176,11 @@ build.photo = function (data) {
 			}
 
 			if (thumb2x !== '') {
-				thumb2x = `srcset='${data.thumbUrl} 200w, ${thumb2x} 400w'`
+				thumb2x = `class="lazyload" src="dist/images/placeholder.png" data-srcset='${data.thumbUrl} 200w, ${thumb2x} 400w'`
```

Comment:
I don't think you need the `class="lazyload"` here because it is added in line 183. :)